package.json

```diff
@@ -48,6 +52,7 @@
     "htmlhint": "^0.10.1",
     "nunjucks": "^3.1.6",
     "sass-lint": "^1.12.1",
+    "start-server-and-test": "^1.7.11",
```

Comment:
Where does this dependency get used? I can't see it run anywhere in the Docker image or within any `package.json` scripts.

Reply:
removed   "start-server-and-test": "^1.7.11",